src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -739,4 +745,22 @@ export default class DefaultAudioVideoController implements AudioVideoController
       this.meetingSessionContext.signalingClient.resume([streamId]);
     }
   }
+
+  getRemoteVideosAvailable(): VideoSendingAttendee[] {
+    const { videoStreamIndex } = this.meetingSessionContext;
+    if (!videoStreamIndex) {
+      this.logger.info('meeting has not started');
+      return [];
+    }
+    const selfAttendeeId = this.configuration.credentials.attendeeId;
+    return videoStreamIndex.allVideoSendingAttendeesExcludingSelf(selfAttendeeId);
```

Comment:
The builder can call this `getRemoteVideosAvailable` API after stopping the session. Note that SDK doesn't clean up the `this.meetingSessionContext` or `this.meetingSessionContext.videoStreamIndex`.

Tip: You can test it using the demo app. Join a meeting, open the browser console, and run these commands:

```
// Stop the session. (If you click on the leave or exit button, the demo will be redirected to the entry page. This is the demo's behavior, not SDK.)
app.audioVideo.stop();

// After a few seconds, check the return value of your method. The expected value is probably `[]`?
app.audioVideo.audioVideoController.getRemoteVideosAvailable()
```

This will be a good test case too!

Reply:
Yes, it returns `[]`, I have also added a unit test for this now. Thanks!